loops-arrays-objects.js

```diff
@@ -31,10 +30,23 @@ const scores = [
 // A
 // D
 // ==========================================
-
-
-
-
+for (let i = 0; i < scores.length; i++) {
+    if(scores[i].score < 60) {
+        console.log("F");
+    }
+    else if (scores[i].score < 70) {
+        console.log("D")
+    }
+    else if (scores[i].score < 80) {
+        console.log("C")
+    }
+    else if (scores[i].score < 90) {
+        console.log("B")
+    }
+    else if (scores[i].score < 100) {
+        console.log("A")
+    }
+}
```

Comment:
Het enige dat ik hier kan bedenken is dat je de code net iets korter had kunnen maken door de else if en de else statements achter } te plaatsen. Zo scheelt het weer een regel of 4 a 5 en dit is ook hoe het "normaal" geschreven wordt.

Reply:
Top ja inderdaad!